actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -1275,24 +1165,6 @@ private void upsertAddress(Map<String, Object> addressReq) {
     cassandraOperation.upsertRecord(orgDbInfo.getKeySpace(), orgDbInfo.getTableName(), addressReq);
   }
 
-  /**
-   * Inserts the Organisation to Organisation relation is not exists, else Updates the Organisation
-   * to Organisation relation
-   */
-  public void upsertOrgMap(
-      String orgId, String parentOrgId, String rootOrgId, int env, String relation) {
-    Util.DbInfo orgMapDbInfo = Util.dbInfoMap.get(JsonKey.ORG_MAP_DB);
-    String orgMapId = ProjectUtil.getUniqueIdFromTimestamp(env);
-    Map<String, Object> orgMap = new HashMap<>();
-    orgMap.put(JsonKey.ID, orgMapId);
-    orgMap.put(JsonKey.ORG_ID_ONE, parentOrgId);
-    orgMap.put(JsonKey.RELATION, relation);
-    orgMap.put(JsonKey.ORG_ID_TWO, orgId);
-    orgMap.put(JsonKey.ROOT_ORG_ID, rootOrgId);
-    cassandraOperation.insertRecord(
-        orgMapDbInfo.getKeySpace(), orgMapDbInfo.getTableName(), orgMap);
-  }
-
   /** Checks whether the parent Organisation exists */
   @SuppressWarnings("unchecked")
   public Boolean isValidParent(String parentId) {
```

Comment:
This method is not in use now, we can delete this

Reply:
done